mappings/net/minecraft/entity/projectile/ProjectileEntity.mapping

```diff
@@ -4,12 +4,20 @@ CLASS net/minecraft/class_1676 net/minecraft/entity/projectile/ProjectileEntity
 	FIELD field_28646 shot Z
 	FIELD field_33399 owner Lnet/minecraft/class_1297;
 	METHOD method_24919 setProperties (Lnet/minecraft/class_1297;FFFFF)V
-		ARG 1 user
+		COMMENT Sets velocity and rotation.
+		ARG 1 shooter
+			COMMENT the entity who shot this projectile; used to add the shooter's velocity
+			COMMENT to this projectile
 		ARG 2 pitch
+			COMMENT the pitch
 		ARG 3 yaw
+			COMMENT the yaw
 		ARG 4 roll
-		ARG 5 modifierZ
-		ARG 6 modifierXYZ
+			COMMENT the roll
+		ARG 5 speed
+			COMMENT the length of the velocity
```

Comment:
just say the speed, if you want to specific mention the velocity will be scaled to have this speed in 3d space than in x-z plane projection